shared/gcp_jupyterlab_shared/handlers.py

```diff
@@ -189,3 +189,31 @@ async def post(self, base64_url):
   async def put(self, base64_url):
     """Proxies the HTTP PUT request."""
     await self._make_request(base64_url, 'PUT', self.request.body)
+
+class ProjectHandler(APIHandler):
```

Comment:
You'll want to make sure these routes get added to https://github.com/angelakuo/jupyter-extensions/blob/721ed550bfb8c32b07d91465ce7f91b468fa140a/shared/gcp_jupyterlab_shared/__init__.py#L40

Reply:
Oh good catch!